gafaelfawr/values.yaml

```diff
@@ -0,0 +1,58 @@
+# Hostname used to construct issuers and URLs.
+# e.g. lsst-lsp-<instance>.ncsa.illinois.edu
+host: ""
+
+# Controls the host for the ingress.  Normally set to the same value as
+# the realm, but leave blank for environments using the wildcard virtual
+# host (such as nublado.lsst.codes).
+ingress:
+  host: ""
+
+# Session length/Token expiration (in minutes)
+session_length: 1440
+
+# Existing PVC for redis claim
+# If empty, redis will use emptydir
+redis_claim: ""
+
+vault_secrets:
+  # secret/k8s_operator/<host>/gafaelfawr
+  path: ""
+
+# <client id from cilogon.org or OIDC provider>
+oauth2_proxy_client_id: "" 
+
+# Default capability required for user-based operations (like generating a token)
+user_capability: "exec:user"
+
+image: "lsstdm/jwt_authorizer:stable"
+
+config:
+  loglevel: DEBUG
+
+  # Defining all capabilites applications may need.
+  # This feeds into the token downlad page
+  known_capabilities:
+    exec:portal: Use the Portal to execute operations. Mostly used from notebook APIs.
```

Comment:
wow, TIL you can do this, I figured YAML would yell about the :

Reply:
I went ahead and added quotes because I found this surprising too.